content/applications/productivity/to_do.rst

```diff
@@ -0,0 +1,148 @@
+
+      .. image:: to_do/fold-edit-delete.png
+         :align: center
+         :alt: Settings button
+
+Creating to-dos
+===============
+
+To create a to-do, click :guilabel:`New` or the **+** button next to a stage name. A new card pops
+up. Add a title to your to-do, then click :guilabel:`Add` to save it, or :guilabel:`Edit` if you
+want to edit it.
+
+.. tip::
+   You can create a new to-do on the fly from any app in Odoo by clicking the clock icon, and
+   :guilabel:`Add a To-Do`:
+
+   .. image:: to_do/add-a-to-do-on-the-fly.png
+      :align: center
+      :alt: Add a to-do on the fly
+
+Editing to-dos
+==============
+
+Different actions are possible in a to-do:
+
+- :guilabel:`Mark as done`: click the check mark button when the task has been completed;
+- :guilabel:`Archive`, :guilabel:`Duplicate`, :guilabel:`Delete`, :guilabel:`Dashboard`, or
+  :guilabel:`Convert to task`: click on the cog icon to see these options;
+- :guilabel:`Tags`: select tags from the dropdown list to categorize your to-do. You can create a
+  new tag simply by writing it;
+- :guilabel:`Assignees`: select one or more users from the dropdown list to assign your to-do;
+- **Customize your tasks content**: when typing / in the text area, a dropdown list of features
+  related to the structure, format, media, navigation, widgets, banner, or basic blocks appear.
+- :ref:`Personal stages <to-do/personal-stages>`: move your task to another stage by clicking on
+  your desired stage.
+
+.. note::
+   To-dos, like private tasks, are only accessible to the users specified as assignees. You can use
+   the :guilabel:`Assignees` field to share a to-do with other users.
+
+.. image:: to_do/task-edition.png
+      :align: center
+      :alt: edit your task
+
+Scheduling activities
+=====================
+
+To help you get things done, you can schedule activities. From a to-do, click on
+:guilabel:`Activities`; or from the dashboard, click the clock icon on a to-do card, then
+:guilabel:`+ schedule an activity`:
+
+.. image:: to_do/schedule-activity.png
+      :align: center
+      :alt: schedule an activity
+
+Then, fill in the required fields:
+
+- :guilabel:`Activity type`: select the type of activity from the list;
+- :guilabel:`Summary`: add a brief description of the activity;
+- :guilabel:`Due date`: select the date the activity is due to;
+- :guilabel:`Assigned to`: select an assignee for the activity;
+- :guilabel:`Log a note`: you can add further description to the activity.
+
+Click :guilabel:`Schedule`.
+
+Convert to-dos into tasks
+=========================
+
+You can convert a to-do into a task and assign it to a project by going to the to-do you want to
+convert, click the cog icon and :guilabel:`Convert to Task`. Select the :guilabel:`Project` you
+want, add :guilabel:`Assignees`, :guilabel:`tags`, and click :guilabel:`Convert to Task`. The to-do
+is now like a private task and appears in the Project app under :guilabel:`My Tasks`.
```

Comment:
You can convert a to-do into a task and assign it to a project. You can do so by going to the to-do you want to convert, clicking the cog icon, and :guilabel:`Convert to Task`. Select the :guilabel:`Project` you want, add :guilabel:`Assignees`, :guilabel:`tags`, and click :guilabel:`Convert to Task`. The to-do is now a private task in the Project app under :guilabel:`My Tasks`.